.travis.yml

```diff
@@ -15,11 +15,10 @@ cache:
  - npm
 language: ruby
 rvm:
-  - jruby
-  - ruby-1.9.3
-  - ruby-2.0.0
-  - ruby-2.1
-  - ruby-2.2
-  - ruby-2.3.0
+  - 2.0.0
+  - 2.1.9
+  - 2.2.5
+  - 2.3.1
+  - jruby-9.0.5.0
```

Comment:
please move `jruby` on top as it takes longest - so the rest can finish when it's running